### PR TITLE
[ICDS Dashboard] reverse sort order in AG Chart page.

### PR DESCRIPTION
**Pull request by Rohit25negi**
Reverse the Sort in adolescent girls page (under FF)
ticket: https://app.asana.com/0/1112385193248823/1145796612945193

### CODE REVIEW

File: custom/icds_reports/reports/adolescent_girls.py [hunk 1]
```diff
@@ -238,7 +238,7 @@ def get_adolescent_girls_data_chart(domain, config, loc_level, show_test=False,
     ]
     all_locations_sorted_by_name = sorted(all_locations, key=lambda x: x['loc_name'])
     all_locations_sorted_by_value_and_name = sorted(
-        all_locations_sorted_by_name, key=lambda x: x['value'], reverse=True)
+        all_locations_sorted_by_name, key=lambda x: -x['value'] if beta else x['value'], reverse=True)
```
Comment (czue) [inline custom/icds_reports/reports/adolescent_girls.py]: two (in my opinion) clearer implementations would be:

1. Flip the condition and remove `reverse=True`.
2. Keep as x['value'] and change `reverse=not beta`.

**Review by czue — APPROVED** [review]
not blocking

File: custom/icds_reports/reports/adolescent_girls.py (hunk 1 above)
Comment (Rohit25negi) [reply custom/icds_reports/reports/adolescent_girls.py]: Damn yeah. you are right! how could I not see it.

**Review by czue — APPROVED** [review]
:+1:

**Pull request merged**